lighthouse-core/gather/driver.js

```diff
@@ -630,6 +630,60 @@ class Driver {
     return new LHElement(targetNode, this);
   }
 
+  /**
+   * @param {string} handle
+   * @param {{timeout: number}=} options,
+   * @return {Promise<string>}
```

Comment:
string is fine for now and any use case I can envision in the future.

if we wanted to fetch image data, this function would need to return a Buffer, and we'd need an option in `fetchResourceOverProtocol` to return data as Buffer or a string (or just return Buffer have the caller deal with it)